common/dns.go

```diff
@@ -0,0 +1,76 @@
+package common
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/giantswarm/clustertest/pkg/application"
+	"github.com/giantswarm/clustertest/pkg/logger"
+	"github.com/giantswarm/clustertest/pkg/wait"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func runDNS() {
+	Context("dns", func() {
+		var (
+			resolver *net.Resolver
+			values   *application.DefaultAppsValues
```

Comment:
Shouldn't this be `ClusterValues` ?

Reply:
Yeah missed it, still worked though :laughing: